test/fixtures/cortege_lineups.yml

```diff
@@ -0,0 +1,22 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
I disagree. The name should however have the 's' somewhere else (all_cortege_lineups).